USAGE.md

```diff
@@ -146,6 +146,62 @@ export default () => {
 };
 ```
 
+## Metro bundler
```

Comment:
`##` makes it sub-header to 
> Use with content loaded from uri

Reply:
We could probably make a `# Web configuration` header and a
`## Metro bundler` + `## Webpack` subheaders